tools/setool/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+package-lock.json
```

Comment:
now that I see this... is package-lock autogenerated from package.json? If it's the case we can ignore it in the whole repository.